### DISCUSSION TITLE
Quick navigation to notebook's dir in jupyterlab

### DISCUSSION THREAD

**Discussion by rilshok** [jupyterlab/jupyterlab#16458] [Ideas]
I have been a dedicated user of JupyterLab for a while now, and it has become an indispensable tool for me. JupyterLab remains my favorite tool, effectively meeting my work needs.

I would like to propose a potentially valuable feature. It would be helpful to have a quick way to switch to the File Browser directory corresponding to the current active notebook tab.

During my work, juggling multiple related tasks simultaneously with several open tabs, I often find that the relevant notebooks are in different directories. When switching to a specific tab, I frequently need to open another notebook located in the same directory. Navigating through the directory hierarchy in the File Browser is convenient for me, but with a high workload and many open tabs, I sometimes get lost and slow down in finding the necessary file.

If there were a shortcut or a button in the graphical interface that could switch the File Browser to the root of the directory where the current active notebook tab is located, I believe it would significantly improve my interaction with JupyterLab.

**Comment by jtpio** [comment]
@rilshok how about right clicking on the notebook tab and choosing "Show in File Browser"?

[jupyterlab-show-in-file-browser.webm](https://github.com/user-attachments/assets/2bfa8128-d5cf-41c2-ab4b-9bfe737ff479)

**Reply by rilshok** [reply]
@jtpio I didn't know this feature existed until today. Thank you!
